programs/vote/src/vote_state/mod.rs

```diff
@@ -972,10 +1039,18 @@ pub fn do_process_vote_state_update(
     )?;
     process_new_vote_state(
         vote_state,
-        vote_state_update.lockouts,
+        vote_state_update
```

Comment:
dereplicode with `lockouts_to_votes()` helper method below (raise it out of the tests module)